db/migrate/20161103221846_remove_invalid_users.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class RemoveInvalidUsers < ActiveRecord::Migration[5.0]
+  class User < ActiveRecord::Base
+  end
+
+  def change
+    User.where(deleted_at: nil, external_id: nil, integration: false).update_all(deleted_at: Time.at(1478211603))
```

Comment:
just picking a random timestamp so I have something unique to query in case things go wrong :D